website/project/model.py

```diff
@@ -2173,6 +2186,7 @@ def fork_node(self, auth, title='Fork of '):
         # Clear permissions before adding users
         forked.permissions = {}
         forked.visible_contributor_ids = []
+        forked.save()
```

Comment:
I mean, I guess we can use a signal to get around that. My instinct would be to move the import inside the method that I presume we'd have on `Node`:

```python
class Node:
    def create_mailing_list_subscription(self, *a, **kw):
        from website.notifications.model import NotificationSubscription
        etc.
```

Then we could call `self.create_mailing_list_subscription` directly instead of using signal indirection  wrapped in an extra `save` call.

At the very least, I'd like to see a comment here explaining why we're saving twice.